### ISSUE TITLE
Using Replicate Loras / Weights ?

### ISSUE THREAD

**Issue by lizaleonekal** [fofr/cog-comfyui#208]
Hi,

Is it possible to use weights that are already available on Replicate as a standalone deployment?

- I know that uploading weights from HF or CivitAI into the input attribute returns an error.
- I've tried inserting the deployment URL of a Flux Lora ([example](https://replicate.com/fpsorg/emoji)) in the Lora loader node, but that didn't work either..

Maybe I'm missing something, but in any case that would be a great feature to have 😅

**Comment by lizaleonekal** [comment]
Bumping this up !

**Comment by FurkanGozukara** [comment]
Yes we also need custom model and lora loading and we have been waiting for weeks for https://www.prompts.ai/

@fofr

**Comment by sourceful-karlson** [comment]
https://github.com/[sourceful-official/ComfyUI_LoadLoraModelOnlyWithUrl](https://github.com/sourceful-official/ComfyUI_LoadLoraModelOnlyWithUrl) 

maybe you can use a node that download the external lora, lora is usually quite small

**Comment by FurkanGozukara** [comment]
> [https://github.com/[sourceful-official/ComfyUI_LoadLoraModelOnlyWithUrl](https://github.com/sourceful-official/ComfyUI_LoadLoraModelOnlyWithUrl)](https://github.com/%5Bsourceful-official/ComfyUI_LoadLoraModelOnlyWithUrl%5D(https://github.com/sourceful-official/ComfyUI_LoadLoraModelOnlyWithUrl))
> 
> maybe you can use a node that download the external lora, lora is usually quite small

which node?

we are using what fofr deployed on replicate is that node available? please give more info if possible

also we want to be able to download base models as well

i don't think it should take this much

today another client asked me this who is also trying to develop another system

i told that replicate best to scale but this feature that you need still lacking @fofr

**Comment by guivr** [comment]
This now works
